packages/uui-tabs/lib/uui-tab-group.element.ts

```diff
@@ -55,7 +66,7 @@ export class UUITabGroupElement extends LitElement {
 
   private onTabActive = (e: MouseEvent) => {
     //? should this contain stopPropagation?
-    const selectedElement: UUITabElement = e.target as UUITabElement;
+    const selectedElement = e.target as UUITabElement;
     selectedElement.active = true;
```

Comment:
Interesting - it makes sense if it's just for styling. I will look into it, then that will also resolve the other conversation on this element if filtering tab nodes.

Reply:
Oh, something came up with this. It's because UUITab extends from ActiveMixin that we do this. I guess this change is a bit out of scope for this feature if we want to change the mixin?

Reply:
I think I found a way here by checking for the active property only when it's needed, so you could build your own tab component that just follows the interface of having an active property and/or extend from ActiveMixin yourself.